modules/services/hound.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.hound;
+
+  configFile = pkgs.writeText "hound-config.json" (
+    builtins.toJSON {
+      inherit (cfg) max-concurrent-indexers dbpath repos;
+      health-check-url = "/healthz";
+    });
+
+  houndOptions = [
+    "--addr ${cfg.addr}"
+    "--conf ${configFile}"
+  ];
+
+  execStart = pkgs.writeScript "houndd-execstart" ''
+    #!${pkgs.runtimeShell}
+    export PATH="${pkgs.mercurial}/bin:${pkgs.git}/bin''${PATH:+:}$PATH"
```

Comment:
I think it might be nicer to have something like
```
Environment = "PATH=${makeBinPath [ pkgs.mercurial pkgs.git ]}";
ExecStart = "${cfg.package}/bin/houndd ${concatStringsSep " " houndOptions}";
```
to the unit configuration.